libs/feat/design-system/components/atoms/Fields/Button/index.d.ts

```
@@ -0,0 +1,12 @@
+import { Button as UiButton } from '@lssm/ui-kit/ui/button';
+import type { ComponentProps } from 'react';
+import type { IconName } from '@/icons-kit/icons';
+interface ButtonProps extends ComponentProps<typeof UiButton> {
+  label: string;
+  startIcon?: IconName;
+  endIcon?: IconName;
+}
+export declare const Button: (
+  props: ButtonProps,
+) => import('react/jsx-runtime').JSX.Element;
+export {};
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)